superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndSelectLabel.tsx

```diff
@@ -43,11 +43,17 @@ export type DndSelectLabelProps = {
   valuesRenderer: () => ReactNode;
   displayGhostButton?: boolean;
   onClickGhostButton?: () => void;
+  /**
+   * Adds a select all button next to the header, allowing the user to 
+   * select all options in one click. Can only be a SelectAllButton or null.  
+   */
+  selectAllButton?: ReactNode;
```

Comment:
```suggestion
  headerAfter?: ReactNode;
```

I think it's okay to make this node more generic. You never know if you want to add a different type of control here. For example, in non-DnD mode adhoc metrics control, this is a plus button.